afs/GettingStarted.md

```diff
@@ -0,0 +1,329 @@
+# Getting started with AFS
+
+**AFS** stands for **A**pplication **F**ile**S**ystem.
+
+An AFS is meant to be used to organize your **business** data and store them,
+like a file system does for plain files.
+
+The structure of an AFS looks like:
+
+```
+   AppData
+     +-- FileSystem1
+     |   +-- File1
+     |   +-- File2
+     |   +-- Project1
+     |   |   +-- RootFolder
+     |   |       +-- ProjectFile1
+     |   |       +-- ProjectFolder1
+     |   |       |   +-- ProjectFile2
+     |   |       +-- ProjectFolder2
+     |   |           +-- ProjectFile3
+     |   +-- Project2
+     |      ...
+     |
+     +-- FileSystem2
+         ...
+```
+where each "project file" may represent a business object, for instance a network or a list of contingencies, or even a computation.
+
+Below, you will learn:
+ - how to **use** the AFS API in your application
+ - how you can **extend** and customize AFS for your needs
+
+# 1- Using AFS
+
+### Using AFS in your java application
+
+In order to use AFS in your application, you will first need to add some dependencies to your project:
+ - `powsybl-afs-core` to use the core API in your code
+ - an actual implementation available at runtime: powsybl comes with `powsyl-afs-mapdb` for prototyping
+ - basic file types defined as extensions of the core: `powsybl-afs-ext-base`.
+
+For instance, if you use maven, in the dependencies section:
+```xml
+<dependency>
+  <groupId>com.powsybl</groupId>
+  <artifactId>powsybl-afs-core</artifactId>
+</dependency>
+<dependency>
+  <groupId>com.powsybl</groupId>
+  <artifactId>powsybl-afs-ext-base</artifactId>
+</dependency>
+<dependency>
+  <groupId>com.powsybl</groupId>
+  <artifactId>powsybl-afs-mapdb</artifactId>
+  <scope>runtime</scope>
+</dependency>
+```
+
+By default, powsybl will load application file system depending on your platform configuration (file *${HOME}/.itools/config.xml*). To tell AFS where it should store its data, you will need to configure your mapdb file system:
```

Comment:
As default configuration is now a Yaml file, it could be interesting to give also an example in yml